src/lib/y2network/connection_config/base.rb

```diff
@@ -204,6 +204,25 @@ def static?
         bootproto ? bootproto.static? : true
       end
 
+      # Return the first hostname associated with the primary IP address.
+      #
+      # @return [String, nil] returns the hostname associated with the primary
+      #   IP address or nil
+      def hostname
+        hostnames&.first
+      end
+
+      # Convenience method in order to modify the canonical hostname mapped to
+      # the primary IP address.
+      #
+      # @param name [String, nil] hostnamme mapped to the primary IP address
```

Comment:
yardoc complains that here is name and in method hname

Reply:
```suggestion
      # @param hname [String, nil] hostnamme mapped to the primary IP address
```